### DISCUSSION TITLE
Is The Github Shop (https://www.thegithubshop.com/) Safe?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#45409] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

I found this site when I was looking for a place to buy Github exclusive shirts, but the site looks very suspicious:

- Copyright and year is still in 2022

- There's no login or registration it takes me straight to the checkout

- But its items are all Just Added (but apparently it's been discontinued since 2022)

Can anyone confirm for me if this site is under the control of Microsoft or Github

**Comment by VladCuciureanu** [comment, marked as answer]
Site is legit. It's been linked to from the official Github account
https://twitter.com/github/status/1427769874625875971

**Reply by ghost** [reply]
Got it, Thank You Very Much

**Reply by VladCuciureanu** [reply]
You're welcome. Enjoy your future swag!
